tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -375,6 +375,7 @@ IncludeByKey.PASSTHROUGH {
 						out_valid_bit_depth	32
 					}
 
+					num_output_pins 2
```

Comment:
we should also clean-up the audio format part @RanderWang, we need to have different structures for in and out. And I also don't get why the format should be the same on all pins?

Reply:
Add clean-up.  @aiChaoSONG  can you help to answer pin format question ?

Reply:
@plbossart both the gain and the smart amp only support 32-bit in this case.